File: programme/volume-of-cone/volume-of-cone.c
// write a programme to find the volume of a cone

#include <stdio.h>

int main(int argc, char *argv[])
{
  double pi = 3.14159;
  double radius, height, volume;
  printf("Enter radius : ");
  scanf("%lf", &radius);
  printf("Enter height : ");
  scanf("%lf", &height);

  volume = pi * radius * radius * height / 3.0;

  printf("\nRadius of cone : %.2lf\n", radius);
  printf("Height of cone : %.2lf\n", height);
  printf("Volume of cone : %.2lf\n", volume);
  return 0;
}

File: programme/volume-of-cone/volume-of-cone.php
// Write a programme to calculate volume of cone

<?php  
$radius = (int)readline('Enter radius: ');  
$height = (int)readline('Enter height: ');
$piediv3 = pi()/3;

$sum = $piediv3*$height*$radius**2;  
echo "Volume of Cone : ". $sum;
?>
